lib/situation_room/check/check_make.ex

```diff
@@ -0,0 +1,25 @@
+defmodule SituationRoom.Check do
+  @moduledoc """
+  Logic for sending requests to site URLs
+  """
+  alias SituationRoom.RequestClient
+
+  def run(url) do
+    start_ms = :os.system_time(:microsecond)
+    res = RequestClient.head(url)
+    time = :os.system_time(:microsecond) - start_ms
+
+    case res do
+      {:ok, response} ->
+        {:ok, %{"status" => response.status, "response_time" => time / 1000}}
+
+      {:error, {reason}} ->
```

Comment:
In what case is this the return tuple vs the pattern on line 19?  Looking at the `Tesla.Env.result()` typing, this pattern doesn't seem to be a possible return type.

Reply:
There were random cases where a tuple was being returned in my tests, so I added this in to pattern match those cases.